app/grandchallenge/evaluation/admin.py

```diff
@@ -58,6 +65,84 @@ def clean(self):
         return cleaned_data
 
 
+class ConfigureAlgorithmPhasesForm(Form):
+    phases = ModelMultipleChoiceField(
+        queryset=Phase.objects.select_related("challenge")
+        .filter(submission_kind=SubmissionKindChoices.CSV)
+        .all()
+    )
```

Comment:
Unsure if we should allow this for multiple phases, it might be a bit safer if we only allow it for individual ones.

Maybe we should exclude phases that have a method and phases that have submissions for more safety?

Reply:
I feel like allowing to do this for multiple phases is one of the main use cases of this form, right? We often have to configure two (or even more phases) in exactly the same way. 

But I agree that exlcuding phases with a method and submissions is a good idea to lock it down more.